base_layer/core/src/consensus/consensus_constants.rs

```diff
@@ -338,7 +338,7 @@ impl ConsensusConstants {
         });
         vec![ConsensusConstants {
             effective_from_height: 0,
-            coinbase_lock_height: 360,
+            coinbase_lock_height: 3,
```

Comment:
?

Reply:
heh, yeah, It's hard to test with 360 ;-)